.github/workflows/build-test.yaml

```
@@ -533,6 +533,117 @@ jobs:
           echo "::notice ::support bundle uploaded to aws replicated-dev account s3://kots-e2e-build-test-support-bundles/${BUNDLE}"
 
 
+  validate-minimal-rbac-override:
+    runs-on: ubuntu-20.04
+    needs: [can-run-ci, build-kots, build-kotsadm, build-kurl-proxy, build-migrations, push-minio, push-postgres]
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version: [v1.20.14-k3s2,v1.21.8-k3s2,v1.22.5-k3s2,v1.23.3-k3s1]
+    steps:
+      - uses: replicatedhq/action-k3s@main
+        id: k3s
+        with:
+          version: ${{ matrix.k8s_version }}
+
+      - name: download kots binary
+        uses: actions/download-artifact@v2
+        with:
+          name: kots
+          path: bin/
+
+      - run: chmod +x bin/kots
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '17.x'
+
+      - name: setup testIM
+        run: npm i -g @testim/testim-cli
+        shell: bash
+
+      ## this is not a testim test
+
+      - name: minimal rbac override on command line
+        env:
+          APP_SLUG: minimal-rbac
+          APP_VERSION_LABEL: "0.0.1"
+        run: |
+          kubectl delete ns $APP_SLUG || true
```

why is this needed? also you can do
```suggestion
          kubectl delete ns $APP_SLUG --ignore-not-found
```